src/components/footer.jsx

```diff
@@ -18,15 +18,15 @@ function Footer() {
 
   const primaryLinks = links.primary;
   const primaryLinksLength = primaryLinks.length;
-  const primaryLinksLabels = t('footer.primaryLinks');
+  const primaryLinksLabels = t('navigation.dropdowns').map((element) => element.simpleLinks).flat().concat(t('navigation.parentLinks'));
   const primaryLinksLabelsLength = primaryLinksLabels.length;
   const primaryLength = primaryLinksLength > primaryLinksLabelsLength
     ? primaryLinksLabelsLength
     : primaryLinksLength;
   // take shorter length if is missing link in primaryLinks or missing label in translation file
-  if (primaryLinksLength !== primaryLinksLabelsLength) {
+  if (primaryLinksLength !== primaryLinksLabelsLength) { // TODO: upto here
```

Comment:
What is the to do here?

Reply:
It was a personal note of where I last left off 🤦🏽‍♂️